webpack.base.config.js

```diff
@@ -50,28 +52,16 @@ let baseConfig = {
       {
         test: /\.s?css$/,
         use: [
-          { loader: 'style-loader' },
-          { loader: 'css-loader' },
-          { loader: 'postcss-loader',
-            options: {
-              plugins: function () {
-                return [ require('postcss-rtl')() ];
-              },
-            },
-          },
-          { loader: 'sass-loader'},
+          'style-loader',
+          'css-loader',
+          'postcss-loader',
+          'sass-loader',
         ],
       },
       {
-        test: /\.(png|svg|gif)$/,
-        use: {
-          loader: 'file-loader',
-          options: {
-            outputPath: 'img/',
-            name: '[name].[ext]',
-          },
-        },
-      },
+        test: /\.(png|jpg|gif|svg)$/,
+        type: 'asset/resource',
+     },
```

Comment:
I was having a lot of issues trying to resolve some weirdness with sass-loader and file-loader interactions. But then I realized that file-loader's functionality is now rolled into webpack itself. This, combiled with `assetModuleFilename: 'img/[name][ext]',` being added to `output` above takes care of things.